src/controller/HomeController.php

```diff
@@ -38,10 +42,10 @@ private function __construct()
      * 
      * @return \App\controller\HomeController
      */
-    public static function getInstance() :HomeController
+    public static function getInstance(TemplateInterface $_templateEngine) :HomeController
```

Comment:
```suggestion
    public static function getInstance(TemplateInterface $templateEngine) :HomeController
```

Le `_ `est uniquement réservé au variable privé d'une classe